libraries/DataFlash/DFMessageWriter.cpp

```diff
@@ -23,6 +23,7 @@ void DFMessageWriter_DFLogStart::reset()
     _fmt_done = false;
     _writesysinfo.reset();
     _writeentiremission.reset();
+    _writeallrallypoints.reset();
```

Comment:
Not new since apparently the earlier lines do it as well, but this is really hard to read... More camelcase or underscores would help....